get logs for failed pods in node e2e tests

mnshaw opened the pull request:
@timstclair 

Initial commit, still work in progress.

stephenrlouie committed:
Refactor common

aledbf committed:
Improve docs

lavalamp committed:
refactor e2e checking

lavalamp committed:
add build cop tools

aledbf committed:
Avoid sync without a reachable master

aledbf committed:
Change annotation name secure-upstreams to secure-backends

aledbf committed:
Update keepalived to 1.2.22

aledbf committed:
Use delayed queue

mnshaw committed:
get logs for failed pods in node e2e tests

mnshaw committed:
get logs for failed pods in node e2e tests

mnshaw committed:
add godep and fix timestamp parsing

bprashanth committed:
Merge pull request #1130 from aledbf/improve-docs

[nginx-ingress-controller] Improve docs and examples

bprashanth committed:
Merge pull request #1253 from aledbf/use-delayed-queue

[nginx-ingress-controller]: Use delayed queue

bprashanth committed:
Merge pull request #1233 from aledbf/avoid-sync

[nginx-ingress-controller]: Avoid sync without a reachable master

bprashanth committed:
Revert "[nginx-ingress-controller]: Avoid sync without a reachable master"

bprashanth committed:
Merge pull request #1256 from kubernetes/revert-1233-avoid-sync

Revert "[nginx-ingress-controller]: Avoid sync without a reachable master"

aledbf committed:
Avoid sync without a reachable master

lavalamp committed:
Merge pull request #1228 from lavalamp/ff4

add build cop tools

lavalamp committed:
fix silly test bug

Commit:
Improve build-cop README.md

lavalamp committed:
Merge pull request #1261 from girishkalele/buildcop1

Improve build-cop README.md

eparis committed:
Merge pull request #1260 from lavalamp/ff4

fix silly test bug

lavalamp committed:
limit number of flakes we keep in memory

eparis committed:
Update godep to include diskcache

eparis committed:
Use the new disk cache

timstclair commented:
I was just introduced to "k8s-gubernator" (e.g. https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/kubernetes-e2e-gce/19087/), which might actually be a better fit for this. I'll follow up with the owner's of that project and see how you might integrate with it. For now, if you want you could take a look at https://github.com/kubernetes/test-infra/tree/master/gubernator

eparis committed:
Merge pull request #1190 from eparis/disk-cache

Write github cache to disk

lavalamp committed:
Merge pull request #1262 from lavalamp/ff5

limit number of flakes we keep in memory

lavalamp committed:
open for every milestone

lavalamp committed:
Merge pull request #1269 from lavalamp/ff4

open for every milestone

lavalamp committed:
fix deployment

lavalamp committed:
Merge pull request #1270 from lavalamp/ff4

fix deployment

eparis committed:
submit-queue: update the 'details' link in github

We were sending people to the front page of the submit-queue. But then
we changed the front page to be the queue instead of the 'all PRs'.

While this link it's particularly useful, at least seeing your PR most
of the time is better than not...

eparis committed:
Merge pull request #1272 from eparis/fix-status-link

submit-queue: update the 'details' link in github

fejta committed:
Switch graph to use svg instead of png

eparis committed:
Merge pull request #1274 from kubernetes/svg

Switch graph to use svg instead of png

bprashanth committed:
Merge pull request #1258 from aledbf/fix-1233

[nginx-ingress-controller]: Avoid sync without a reachable master

Commit:
Add soak tests to the dashboard

lavalamp committed:
Merge pull request #1277 from krousey/add_soak

Add soak tests to the dashboard

mwielgus committed:
Cluster-autoscaler: fix for multi-mig autoscaling

mwielgus committed:
Merge pull request #1282 from mwielgus/mig-node

Cluster-autoscaler: fix for multi-mig autoscaling

ingvagabund committed:
Merge pull request #1001 from stephenrlouie/common_vars

refactor common

mnshaw committed:
fix conflicts

mnshaw committed:
add flags for url of bucket and filters for kubelet log

eparis commented:
this PR seems pretty mis-formed?   43 commits from lots of different people across most of contrib...  Can you clean it up to only be the single change you are hoping to make?
